desktop/apps/consignments2/client/actions.js

```diff
@@ -14,12 +35,42 @@ export function createSubmission () {
   }
 }
 
+export function fetchArtistSuggestions (value) {
+  return async (dispatch, getState) => {
+    try {
+      request.get(`${sd.API_URL}/api/v1/match/artists`)
+        .query({ visible_to_public: 'true', term: value })
+        .set('X-ACCESS-TOKEN', sd.CURRENT_USER.accessToken)
+        .end((err, res) => {
+          if (!err || res.ok) {
+            dispatch(updateArtistSuggestions(res.body))
+            dispatch(hideNotConsigningMessage())
+          }
+        })
+    } catch (error) {
+      console.error('error!', error)
```

Comment:
Ah, gotcha. Yea- I guess that's more of a UI or app-by-app concern. I think at some point we had a purple banner at the top you _could_ have used, but we sort of stopped that pattern. /shrug